nengo_viz/static/viz_sim_control.js

```diff
@@ -37,15 +37,19 @@ VIZ.SimControl = function(div, args) {
     
     /** Create the pause button */
     this.pause_button = document.createElement('button');
-    this.pause_button.innerHTML='Pause';
+    this.pause_button.className = "btn btn-default play-pause-button";
+    this.pause_button.innerHTML='<i class="glyphicon glyphicon-pause"></i>';
```

Comment:
Weird...  although according to that page, Bootstrap 3 recommends using ```<span>``` instead of ```<i>```, which feels better to me....